mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -135,121 +198,172 @@ export class CompareRunView extends Component {
-                <tr>
-                  <th
-                    scope='rowgroup'
-                    className='inter-title'
-                    colSpan={this.props.runInfos.length + 1}
-                  >
-                    <h2>
-                      <FormattedMessage
-                        defaultMessage='Metrics'
-                        // eslint-disable-next-line max-len
-                        description='Row group title for metrics of runs on the experiment compare runs page'
-                      />
-                    </h2>
-                  </th>
-                </tr>
-                {this.renderDataRows(
-                  this.props.metricLists,
-                  false,
-                  (key, data) => {
-                    return (
-                      <Link
-                        to={Routes.getMetricPageRoute(
-                          this.props.runInfos
-                            .map((info) => info.run_uuid)
-                            .filter((uuid, idx) => data[idx] !== undefined),
-                          key,
-                          experimentId,
-                        )}
-                        title='Plot chart'
+                        </Tooltip>
+                      </div>
+                    </td>
+                  );
+                })}
+              </tr>
+              <tr>
+                <th scope='row' className='head-value sticky-header' style={colWidthStyle}>
+                  <FormattedMessage
+                    defaultMessage='Start Time:'
+                    // eslint-disable-next-line max-len
+                    description='Row title for the start time of runs on the experiment compare runs page'
+                  />
+                </th>
+                {this.props.runInfos.map((run) => {
+                  const startTime = run.getStartTime()
+                    ? Utils.formatTimestamp(run.getStartTime())
+                    : '(unknown)';
+                  return (
+                    <td className='data-value' key={run.run_uuid} style={colWidthStyle}>
+                      <Tooltip
+                        title={startTime}
+                        color='blue'
+                        overlayStyle={{ maxWidth: '400px' }}
+                        mouseEnterDelay={1.0}
                       >
-                        {key}
-                        <i className='fas fa-chart-line' style={{ paddingLeft: '6px' }} />
-                      </Link>
-                    );
-                  },
-                  Utils.formatMetric,
-                )}
-              </tbody>
-            </table>
-          </div>
+                        {startTime}
+                      </Tooltip>
+                    </td>
+                  );
+                })}
+              </tr>
+            </tbody>
+          </table>
+        </CollapsibleSection>
+        <CollapsibleSection
+          title={
+            <h1>
```

Comment:
```suggestion
            <h1 style={{ marginTop: 0, marginBottom: 0 }}>
```

Can we remove vertical margins to align the arrow icon and title?

BEFORE

<img width="335" alt="Screen Shot 2022-01-31 at 22 53 03" src="https://user-images.githubusercontent.com/17039389/151806132-f44b34db-e0c1-4a93-b20a-6232bb856325.png">

AFTER

<img width="335" alt="Screen Shot 2022-01-31 at 22 52 40" src="https://user-images.githubusercontent.com/17039389/151806160-39d916e6-9cbc-4404-baa2-8d6019368658.png">